src/main/java/ladder/domain/Person.java

```diff
@@ -0,0 +1,25 @@
+package ladder.domain;
+
+public class Person {
+    public static final int MAX_NAME_LENGTH = 5;
+    private String name;
+
+    private Person(String name) {
+        checkNameLength(name);
+        this.name = name;
+    }
+
+    public static Person create(String name) {
+        return new Person(name);
+    }
+
+    public String tellName(){
```

Comment:
```suggestion
    public String getName(){
```
getter를 사용하지 말라는 의미는 메소드의 네이밍이 getXXX를 사용하면 안된다는것이 아닙니다 ! 
객체 내부의 상태를 꺼내어 연산을 하는 로직을 작성하지 않고 상태를 가지고 있는 객체가 상태를 사용한 연산을 메소드로 제공하여 외부 클래스에서는 해당 객체의 메소드 호출을 사용하라는 의미라고 생각해주시면 좋을것 같아요. 
상태값을 뿌려주거나 연산을 하지 않는 경우에는 getter를 사용하게 될수 밖에 없답니다 ! 
아래 문서가 도움이 되실것 같아요 🙏 
https://woowacourse.github.io/javable/2020-04-28/ask-instead-of-getter